package.json

```diff
@@ -45,6 +51,7 @@
   "dependencies": {
     "bip39": "^2.5.0",
     "niceware": "^1.0.5",
+    "optional-dev-dependency": "^2.0.1",
```

Comment:
does this work if this is in devDependencies instead of dependencies?

Reply:
yes - it absolutely should 😄

Reply:
@diracdeltas moved! (refresh needed since I did a force push 😄)